wandb/sdk/wandb_init.py

```diff
@@ -609,29 +602,18 @@ def init(self) -> Union[Run, RunDisabled, None]:  # noqa: C901
         mailbox = Mailbox()
         backend = Backend(settings=self.settings, manager=manager, mailbox=mailbox)
         backend.ensure_launched()
-        backend.server_connect()
```

Comment:
if you are going to remove this (which i support), remove it in attach function and remove it from backend.py please

Reply:
good point! will do!